tests/Benchmarks/System.Memory/IndexOf.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Numerics;
+using BenchmarkDotNet.Attributes;
+
+namespace System.Memory.Benchmarks
+{
+    public class IndexOf
+    {
+        static byte[] s_buffer = new byte[2000];
+
+        [GlobalSetup]
+        public void PrintInfo() => Console.WriteLine($"// Vector.IsHardwareAccelerated={Vector.IsHardwareAccelerated}"); // just print the info to help understand the results
+
+        [Benchmark(OperationsPerInvoke = 6 * 4)] // avoid loop overhead, do manuall loop unrolling
+        [Arguments(0)] // we don't have more arguments here because the CPU cache gets warmed-up very soon and the results are the same for every $at
+        [Arguments(1000)]
+        public int SpanIndexOf(int at)
```

Comment:
The previous test was measuring the IndexOf method and comparing the vectorized vs. non-vectorized, not the span indexer.

Also, do we have a way to programmatically set Vector.IsHardwareAccelerated to true/false? I know it can be done by setting an environment variable before running the test.

Reply:
@ahsonkhan yes, it's possible. Could you tell me the name of env var which controlls that? I will add sample config then

Reply:
You can disable vectorization by setting `COMPlus_FeatureSIMD=0`